jumpscale/packages/vdc_dashboard/chats/etcd.py

```diff
@@ -69,17 +70,56 @@ def initializing(self):
             self.k8s_client.delete_deployed_release(self.release_name)
             self.stop(dedent(stop_message))
 
+        self.is_certified = False
+        start_request_time = time()
```

Comment:
maybe we can replace this with to be more readable
```suggestion
        start_request_time = j.data.time.now().timestamp
```